CHANGELOG.md

```diff
@@ -4,9 +4,13 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+
+## Release v3.1.0 (2024-06-21)
+
 ### Changed
 
-- updated example config
+- The user guide has been restructured [#367](https://github.com/WAM2layers/WAM2layers/pull/367)
+- The example config has been updated [#371](https://github.com/WAM2layers/WAM2layers/pull/371)
```

Comment:
Do you want to add that the preprocessing has been restructured to allow for easier implementation of new datasets?

Reply:
Sure I can add that. It's mostly an internal change that end-users don't notice but it doesn't hurt adding it.